Gems/Multiplayer/Code/Source/NetworkEntity/EntityReplication/PropertyPublisher.cpp

```diff
@@ -21,6 +21,13 @@ namespace Multiplayer
         , m_pendingRecord(remoteNetworkRole)
         , m_sentRecords(net_EntityReplicatorRecordsMax)
     {
+        if ( ownsLifetime == OwnsLifetime::False )
+        {
+            // This entity is owned by some other authority, this publisher will only be used for updating (not creating).
```

Comment:
might want to update this comment to specifically call out that since this replicator does not own it's lifetime, the remote replicator must exist (otherwise, we would never have created a replicator that doesn't own its lifetime).

Reply:
fixed!